src/openrct2/interface/viewport.c

```diff
@@ -1236,21 +1221,19 @@ static bool sub_679074(rct_drawpixelinfo *dpi, sint32 imageId, sint16 x, sint16
         if (xEndPoint <= 0) {
             return false;
         }
-
-        if (!(image->flags & G1_FLAG_RLE_COMPRESSION)) {
-            _unk9ABDAE += x;
-        }
     }
 
     if (image->flags & G1_FLAG_RLE_COMPRESSION) {
         return sub_67933B_679788_679C4A_67A117(image->offset, xStartPoint, yStartPoint, round);
     }
 
     uint8 *offset = image->offset + (yStartPoint * image->width) + xStartPoint;
-    uint32 ebx = gUnkEDF81C;
+    uint32 imageType = gCurrentImageType;
 
-    if (!(image->flags & 2)) {
-        return sub_679236_679662_679B0D_679FF1(ebx, image, offset);
+    if (!(image->flags & G1_FLAG_1)) {
+        // Using assert here because it is probably unused, if not triggered remove this block.
+        assert(false);
```

Comment:
I think you miss understood me. This bit of code is perfectly fine its the block below this that starts with the comment `The code below is untested.` that is the dead code. I've also made a small mistake with the name when i discussed this on gitter. `sub_679236_679662_679B0D_679FF1` is the bmp `pixel_is_present` function and `sub_67933B_679788_679C4A_67A117` is the rle `pixel_is_present` function.